tests/components/deconz/test_services.py

```diff
@@ -259,18 +251,23 @@ async def test_service_refresh_devices_trigger_no_state_update(
                 "uniqueid": "00:00:00:00:00:00:00:01-00",
             }
         }
-    }
-    with patch.dict(DECONZ_WEB_REQUEST, data):
-        config_entry = await setup_deconz_integration(hass, aioclient_mock)
-
+    ],
+)
+@pytest.mark.usefixtures("config_entry_setup")
+async def test_service_refresh_devices_trigger_no_state_update(
+    hass: HomeAssistant,
+    aioclient_mock: AiohttpClientMocker,
+    deconz_payload: dict[str, Any],
+    mock_requests,
```

Comment:
If order doesn't matter here you could use `@pytest.mark.usefixtures` for `mock_requests`

Reply:
nevermind, you call it as `mock_requests()`